e2e/wdio/core/pages/multi-input.po.ts

```diff
@@ -16,12 +16,12 @@ export class MultiInputPo extends CoreBaseComponentPo {
     approveButton = '.fd-button--emphasized';
     multiSelectButton = '.custom-multi-input-select-all-bar-element button';
     mobileInputOptions = 'fd-multi-input-mobile-example .fd-token__text span';
-    displayObjectOptions = 'div#background-ex3 .fd-token__text span';
-    searchTermOptions = 'div#background-ex4 .fd-token__text span';
-    customFilterOptions = 'div#background-ex5 .fd-token__text span';
+    displayObjectOptions = 'fd-multi-input-displaywith-example .fd-token__text span';
```

Comment:
missed a `-` in `displaywith`

Reply:
@InnaAtanasova it's the actual selector of the component. Do you want me to rename it?
<img width="434" alt="image" src="https://user-images.githubusercontent.com/33101123/169327269-3689b64e-d392-449e-83a6-f7ae54d7f855.png">

Reply:
no, it's ok. Leave it as it is. Thanks for replying.